hll/include/HllArray-internal.hpp

```diff
@@ -26,11 +26,11 @@
 #include "CubicInterpolation.hpp"
 #include "CompositeInterpolationXTable.hpp"
 #include "CouponList.hpp"
-
 #include <cstring>
 #include <cmath>
 #include <stdexcept>
 #include <string>
+#include "../../common/include/inv_pow2_table.hpp"
```

Comment:
i think convention is to put project-internal files first?

Reply:
I am not aware of this convention. Perhaps we should discuss.